snapcraft/storeapi/__init__.py

```diff
@@ -97,7 +97,8 @@ def __init__(self, conf, root_url):
         self.session.mount('https://', HTTPAdapter(max_retries=5))
 
         self._snapcraft_headers = {
-            'X-SNAPCRAFT-VERSION': snapcraft.__version__
+            'X-SNAPCRAFT-VERSION': snapcraft.__version__,
+            'User-Agent': 'snapcraft/{}'.format(snapcraft.__version__),
```

Comment:
what is 'testing'?  Do I need to check if snapcraft is installed via a deb package, a snap?